BattleNetwork/bnMoveEvent.cpp

```diff
@@ -40,7 +40,7 @@ bool MoveAction::IsJumping() const
 //!< helper function true if sliding
 bool MoveAction::IsSliding() const 
 {
-  return data.dest && data.deltaFrames > frames(0) && data.height <= 0.0f;
+  return data.dest && data.deltaFrames > frames(0) && (+data.height) <= 0.0f;
```

Comment:
`<=` -> `==` too. This way we're just looking at `eax` is `0` at the ISA level and dont regard any other register state flags to check.